### ISSUE TITLE
Patreon feed support

### ISSUE THREAD

**Issue by bcye** [1Feed/feedback#64]
Support for patreon posts/feeds (not audio feeds)

**Comment by Booligoosh** [comment]
Good idea, I'll look into this! Apparently Patreon has an unofficial API no may be able to use that.
